setup.py

```diff
@@ -69,6 +76,7 @@
     'author_email': about['__email__'],
     'classifiers': CLASSIFIERS,
     'description': about['__summary__'],
+    'extras_require': EXTRAS_REQUIRES,
```

Comment:
Just nitpicking: can you call it `EXTRAS_REQUIRE` just to be consistent?

Reply:
👍  - I had the key wrong at first and went back and changed it. Will fix - thanks.